unlock-app/src/components/interface/keychain/KeyDetails.tsx

```diff
@@ -92,41 +76,76 @@ export const KeysByNetwork = ({ account, network }: KeysByNetworkProps) => {
 export const KeyDetails = () => {
   const { account, network } = useContext(AuthenticationContext)
 
+  const networkItems: any[] =
+    Object.entries(networks ?? {})
```

Comment:
It looks like we do that in many places. I wonder if we should create a util/helper?

Reply:
yes! we can set an helper for that, ill open a specific PR for it